README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Motivation
 
-- Enable easy integration of different components
+- Enable easy integration of different integration
```

Comment:
I think better would be to use `integration of Integrations` (capitalised **I**)

Reply:
similarly, use of `Integration` at places where it is referred as a noun.

Reply:
why not use a synonym to prevent "integration" being used twice?